.github/dependabot.yaml

```diff
@@ -0,0 +1,11 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
suggestion: change the frequency for `github-actions` to `weekly`

Avoid Dependabot spammage.